modules/VersionInfo.cmake

```diff
@@ -21,7 +21,11 @@ else()
 	set( BUILD_VER 0 )
 endif()
 if(NOT BUILD_DATE)
-	string(TIMESTAMP BUILD_DATE "%d.%m.%Y")
+	if (DEFINED ENV{SOURCE_DATE_EPOCH})
+		execute_process(COMMAND "date" "-u" "-d" "@$ENV{SOURCE_DATE_EPOCH}" "+%d.%m.%Y" OUTPUT_VARIABLE BUILD_DATE OUTPUT_STRIP_TRAILING_WHITESPACE)
```

Comment:
In fact, I agree, that must be sufficient. It wasn’t obvious to me I could do that.